app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const logger = require('morgan');
+const bodyParser = require('body-parser');
+process.env.SECRET_KEY = 'superSecretString';
+const app = express();
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+
+require('./server/routes/documents')(app);
```

Comment:
**Code Organization - Nitpick**

- Why not do route related stuff in a `/routes` directory?